CRM/Bsd/Page/BSD.php

```diff
@@ -168,69 +183,109 @@ public function share($param) {
   public function createContact($param) {
     $h = $param->cons_hash;
 
-    $apiAddressGet = 'api.Address.get';
-    $apiAddressCreate = 'api.Address.create';
-    $apiGroupContactGet = 'api.GroupContact.get';
-    $apiGroupContactCreate = 'api.GroupContact.create';
-
     $contact = array(
```

Comment:
aah,

so if you want all addresses, api chaining and call api.address.get is the right way.

if you want only the main address, try return=country or postalcode, it _should_ return the address_id too